arquillian/appclient/src/main/java/tck/arquillian/protocol/appclient/AppClientMethodExecutor.java

```diff
@@ -117,6 +119,7 @@ public TestResult invoke(TestMethodExecutor testMethodExecutor) {
                     break;
             }
             result.addDescription(description);
+            log.info(result.toString() + ": " + description + ": " + vehicleArchiveName + ": " + appArchiveName);
```

Comment:
I think this might look a little better with something like:
```
log.log(level, String.format("%s_%s [%s] %s : %s", vehicaleArchiveName, appArchiveName, result.getStatus(), description, result);
```

Note I'm not sure how the `vehicaleArchiveName` and `appArchiveName` should be formatted.

The `level` should be set in the in the switch statement right above this log line IMO. Then for `ERROR` and `FAILED` status we could set the level to `Level.SEVERE`.

Reply:
Thanks, switching over to this style.  I don't think the appclient jar or vehicle names are what I really wanted to show here.  Showing the EAR name would be nicer (based on how we current include more information in the ear name).  Revising...

Reply:
Will update this to show the new output with:
```
            result.addDescription(description);
            log.log(Level.INFO, String.format("[%s] %s : %s", result.getStatus(), description, result));

```